src/components/TreeGrid.ts

```diff
@@ -82,10 +82,13 @@ type TreeGridProps<T extends RowContents> = {
   readonly getChildren: (row: T) => Promise<T[]>;
   /** Optional header style */
   readonly headerStyle?: CSSProperties;
+  /** Optional opened concept ID for determining initial row */
+  readonly openedConceptId?: number;
```

Comment:
`TreeGrid` is general purpose component and doesn't know about concept IDs. Instead this is a row id (see `RowContents`). Also I wouldn't use `opened` here since the shown row may or may not be "open" in the tree grid sense (and in the cohort builder's case, is not open).
```suggestion
  /** If set, initially scroll this row id into view. */
  readonly scrollToShowRowId?: number;
```